grid/social-media/publish-post/profile.supr

```diff
@@ -43,6 +43,10 @@ usecase PublishPost unsafe {
     media [
       MediaItem
     ]
+
+    attachments [
+      MediaContainer
```

Comment:
I think MediaContainer is a term specific to Instagram, consider using naming the model like, for example, Attachment.
On the other hand, is it necessary to wrap the ID into an object? Since we need only an ID, consider changing this to just [string]. 

Also, you need to bump the minor version of the profile.

Reply:
>I think MediaContainer is a term specific to Instagram, consider using naming the model like, for example, Attachment.

Attachment doesn't have the right connotations in my mind (but I'm open to changing it), is the issue outright *avoiding* provider-specific terms?

>On the other hand, is it necessary to wrap the ID into an object?

Pinterest, for ex. also includes a `cover_image_url` *at the moment of publishing*, and I can imagine other attributes of this kind, so I was trying to be future-proof

 >Also, you need to bump the minor version of the profile.

:+1:

Reply:
> Attachment doesn't have the right connotations in my mind (but I'm open to changing it), is the issue outright avoiding provider-specific terms?

Yes. Also the name isn't related to the field and yet isn't used elswhere. Not sure what the correct term should be here. Maybe use "assets" for the field and "UploadedAssets" for the model to align this with the other profile?

> Pinterest, for ex. also includes a cover_image_url at the moment of publishing, and I can imagine other attributes of this kind, so I was trying to be future-proof

OK, make sense, though I am curious whether we can make an union out of it, like `[string | MediaContainer]`

Reply:
> Also, you need to bump the minor version of the profile.
Welp, should I bump versions of all maps then?